packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-drive-editor/components/CreateDocument.esm.t

```diff
@@ -1,5 +1,5 @@
 ---
-to: "<%= rootDir %>/<%= h.changeCase.param(name) %>/components/CreateDocument.tsx"
+to: "<%= driveEditorDir %>/components/CreateDocument.tsx"
```

Comment:
prefer to use the existing dir name for a given editor, so like editors/some-editor. Otherwise we can accidentally end up creating a new one if the dir name is different from the editor name.